UDPClientGUI/IClientInterface.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPCommGUI
+{
+    interface IClientInterface : IDisposable
+    {
+        IPAddress ConnectionIP { get; set; }
+
+        int ConnectionPort { get; set; }
+
+        string InputMessage { get; set; }
+
+        bool ConnectionStatus { get; }
+
+        void Connect();
+
+        void Disconnect();
+
+        void SendMessage();
+
```

Comment:
Minor thing, watch the spacing (we'll sort this out using Style rules later on).

Reply:
Fixed.